shim/src/repair/objectAccessors.js

```diff
@@ -8,7 +8,9 @@ import { getPrototypeOf, defineProperty, getOwnPropertyDescriptor } from '../com
 // we need to redefine them while in strict mode.
 // https://tc39.github.io/ecma262/#sec-object.prototype.__defineGetter__
 
-export default function repairAccessors(objProto) {
+export default function repairObjectAccessors(confinedGlobal) {
+
+    const { Object: { prototype: objProto } } = confinedGlobal;
```

Comment:
I'll have all repair functions use the same arguments, the global, but be more specific about what they do.